consensus/ibft/src/integration-test/java/tech/pegasys/pantheon/consensus/ibft/support/NodeParams.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.pantheon.consensus.ibft.support;
+
+import tech.pegasys.pantheon.crypto.SECP256K1.KeyPair;
+import tech.pegasys.pantheon.ethereum.core.Address;
+
+public class NodeParams {
+  private final Address address;
+  private final KeyPair nodeKeys;
+
+  public NodeParams(final Address address, final KeyPair nodeKeys) {
+    this.address = address;
+    this.nodeKeys = nodeKeys;
+  }
+
+  public Address getAddress() {
+    return address;
+  }
+
+  public KeyPair getNodeKeys() {
```

Comment:
nit: `getNodeKeyPair` would be more accurate here.  Sounds like the node has multiple key pairs otherwise.

Reply:
done.